### PR TITLE
Jy/allow task

### PR DESCRIPTION
**Pull request by JeeemJeerms**
Add the option to opt into creating a service. The safety check means current configurations aren't affected but new configurations can opt out

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -111,26 +111,29 @@ class ServerlessFargateTasks {
...
-          'NetworkConfiguration': {
-            'AwsvpcConfiguration': Object.assign({
-              'AssignPublicIp': options.vpc['public-ip'] || "DISABLED",
-              'SecurityGroups': options.vpc['security-groups'] || [],
-              'Subnets': options.vpc['subnets'] || [],
-            }, network_override),
-          }
-        }, service_override)
+      // check for optional service if set
+      var allowService = options.tasks[identifier]['allowService'] || true
```
Comment (kbrownlees) [inline lib/index.js]: @JeeemJeerms `|| true` doesn't seem right here?  since `false || true` will always be true?

Comment (JeeemJeerms) [reply lib/index.js]: this is just assigning the variable, so if the variable doesn't exist it will default to true

Comment (kbrownlees) [reply lib/index.js]: @JeeemJeerms it is an OR statement. If the first one is false it will always be the second one:

```
> false || true
true
```

Comment (JeeemJeerms) [reply lib/index.js]: fixed

**Review by kbrownlees — COMMENTED** [review]
@JeeemJeerms where is the formatting coming from? Prettier default I assume?

**Comment by JeeemJeerms** [comment]
@kbrownlees yeah it was, I've just removed it

**Review by kbrownlees — APPROVED** [review]
This will need a manual version bump & upload I believe. I suspect you have permissions to do this?

**Comment by JeeemJeerms** [comment]
@kbrownlees yes I had permissions and published it

**Comment by kbrownlees** [comment]
@JeeemJeerms without a version bump?

**Pull request merged**